dist/s_all

```diff
@@ -51,34 +55,49 @@ test "$reconf" -eq 0 || {
 	run "sh ./reconf" "Rebuilding GNU tools library support")
 }
 
-run "sh ./s_readme $force" "building README file"
-run "python api_config.py" "building WiredTiger API"
-run "python api_err.py" "building WiredTiger error returns"
-run "python flags.py" "building flags"
-run "python log.py" "building logging layer"
-run "python stat.py" "building statistics support"
-run "python java_doc.py" "building Java documentation index"
-run "sh ./s_typedef -b" "building standard typedefs"
-run "sh ./s_prototypes" "building function prototypes"
-run "sh ./s_tags" "building tags files"
+# Non parallelizable commands
+run "sh ./s_readme $force"
+run "python api_config.py"
+run "python api_err.py"
+run "python flags.py"
+run "python log.py"
+run "python stat.py"
+run "python java_doc.py"
+run "sh ./s_prototypes"
+run "sh ./s_typedef -b"
 
-run "sh ./s_copyright" "checking copyright notices"
-run "sh ./s_define" "checking for unused #defines"
-run "sh ./s_typedef -c" "checking for unused typedefs"
-run "sh ./s_funcs" "checking for unused functions"
-run "sh ./s_export" "checking external symbol names"
-run "sh ./s_getopt" "checking for incorrect getopt usage"
-run "sh ./s_label" "checking error/return labels"
-run "sh ./s_lang" "checking for SWIG generated name conflicts"
-run "sh ./s_longlines" "checking for long lines"
-run "sh ./s_stat" "checking for unused statistics fields"
-run "sh ./s_string" "checking string spelling"
-run "python style.py" "checking style (pass 1)"
-run "sh ./s_style" "checking style (pass 2)"
-run "sh ./s_python" "checking Python style"
-run "sh ./s_whitespace" "checking whitespace"
-run "sh ./s_win" "checking windows config"
+read -d '' COMMANDS <<EOM
+'sh ./s_tags > ${t_pfx}tags'
+'sh ./s_copyright > ${t_pfx}s_copyright'
+'sh ./s_define > ${t_pfx}s_define'
+'sh ./s_typedef -c > ${t_pfx}s_typedef_c'
+'sh ./s_funcs > ${t_pfx}s_funcs'
+'sh ./s_export > ${t_pfx}s_export'
+'sh ./s_getopt > ${t_pfx}s_getopt'
+'sh ./s_label > ${t_pfx}s_label'
+'sh ./s_lang > ${t_pfx}s_lang'
+'sh ./s_longlines > ${t_pfx}s_longlines'
+'sh ./s_stat > ${t_pfx}_tat'
+'sh ./s_string > ${t_pfx}s_string'
+'python style.py > ${t_pfx}py_style'
+'sh ./s_style > ${t_pfx}s_style'
```

Comment:
Since s_style is multi-process internally, it seems a little odd to run it in parallel here: now we're up to ~40 processes running in parallel, and if we do the same to any more of these scripts, that number will grow.  Maybe s_style should be run serially, after the other stages?